publisher/start.sh

```diff
@@ -28,33 +28,44 @@ EORESTORE
     docker push ${DESTINATION_DOCKER_IMAGE}-roles-only
 
     docker start publishing_db_container
-    docker exec publishing_db_container cp /files/${backup_file} /dumps/db.backup
-
-    cat <<-EORESTORE | (docker exec -i publishing_db_container sh -c "cat >> restore.sh")
-      pg_restore /dumps/db.backup -U postgres -d ${DB_NAME}
-EORESTORE
 
     # we need to sanitize sensitive data first
-    docker run --name sanitizing_db_container -d --rm -v ${DUMPS_DIR}:/files \
-    -e POSTGRES_DB=${DB_NAME} -e POSTGRES_PASSWORD= postgres:10.5
-    # wait for "database system is ready to accept connections", hoping 4 sec is enough
-    sleep 4;
+    docker run --name sanitizing_db_container -d --rm \
+    -v ${DUMPS_DIR}:/files -v ${SQL_DIR}:/sql \
+    -e POSTGRES_DB=${DB_NAME} -e POSTGRES_PASSWORD= ${POSTGRES_IMAGE}
```

Comment:
can't space here be a problem?
```suggestion
    -e POSTGRES_DB=${DB_NAME} -e POSTGRES_PASSWORD=${POSTGRES_IMAGE}
```

Reply:
The point here is to have empty password. And ${POSTGRES_IMAGE} is image to run.

Reply:
oh, I see now. Maybe add `""`?